esphome/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_ESPHOME_IP
+      APP_PORT: 6052
+      PROXY_AUTH_ADD: "true"
+
+  server:
+    image: esphome/esphome:2022.6.2@sha256:69d382af56b10006a833eb90d29339b12efdddfb93382764a379c17fe252eda6
+    volumes:
+      - ${APP_DATA_DIR}/data:/config
+      - /etc/localtime:/etc/localtime:ro
```

Comment:
Out of interest, how come this gets mounted in?

Reply:
Never really checked tbh.
Just saw that ESPHome's recommended docker-compose had it in

https://esphome.io/guides/getting_started_command_line.html